aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ScheduledV2Event.java

```diff
@@ -0,0 +1,358 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazonaws.services.lambda.runtime.events;
+
+import org.joda.time.DateTime;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * represents a scheduled V2 event
+ */
+public class ScheduledV2Event implements Serializable, Cloneable {
+
+    private static final long serialVersionUID = -463442139623175611L;
+
+    private String version;
+
+    private String account;
+
+    private String region;
+
+    private String detail;
```

Comment:
I think we (AWS) needs to validate that there will be no events where detail will be anything other than a String for scheduled task use cases.